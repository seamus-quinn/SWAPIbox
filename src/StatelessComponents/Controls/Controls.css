.controls {
  background-color: #c29991;
  margin: 5px;
  border-radius: 15px;
}