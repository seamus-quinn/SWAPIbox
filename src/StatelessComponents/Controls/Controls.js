import React from 'react';

const Controls = ({ favorites }) => {
  return(
    <div>
      <p>Favorites {favorites}</p>
      <button>PEOPLE</button>
      <button>PLANETS</button>
      <button>VEHICLES</button>
    </div>
  )
}

export default Controls;
