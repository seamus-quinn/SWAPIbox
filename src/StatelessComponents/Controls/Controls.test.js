import React from 'react';
import ReactDOM from 'react-dom';
import Controls from './Controls'
import { shallow } from 'enzyme';

describe('Controls', () => {

  let wrapper;
  let mockProps;

  beforeEach(() => {
    mockProps = {
      favorites: [ {name: 'Ron', homePlanet: 'Great Briatin' } ],
      setPeopleData: jest.fn(),
      setVehicleData: jest.fn(),
      setPlanetData: jest.fn(),
      displayFavorites: jest.fn()
    }
    wrapper = shallow(<Controls />)
  })

  it('should exist match the snapshot', () => {

    expect(wrapper).toMatchSnapshot();
  })
})