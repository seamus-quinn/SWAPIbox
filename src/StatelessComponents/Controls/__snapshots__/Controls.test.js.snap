// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls should exist match the snapshot 1`] = `undefined`;
